src/plone/app/multilingualindexes/querymodifiers.py

```diff
@@ -15,16 +15,13 @@
 }
 
 
-def _index_of_criteria(query, name):
-    for idx, criteria in enumerate(query):
-        if criteria["i"] == name:
-            return idx
+def _indices_of_criteria(query, name):
+    return [idx for idx, criteria in enumerate(query) if criteria["i"] == name]
```

Comment:
This returns a list, before it returned an int (or None).

Reply:
This change is necessary since we want to allow for multi-path queries the number of used criteria can be 0, 1 or more